INSTALL.md

```diff
@@ -8,6 +8,16 @@ The build system requires the use of the Metrowerk C Compiler 2.0/base to compil
 
 In the future, a GCC option will be available so MWCC is not required to build, however it is required for a matching ROM.
 
-#### 3. Build ROM
+#### 3. Dependencies
+
+Building the ROM requires the following packages:
+
+* make
+* git
+* build-essentials
```

Comment:
These dependencies need to be clearer as they do not match the respective names on all package managers.

Reply:
Those are the debian ones and I'm not adding Cygwin/msys right now.

Reply:
What about e.g. Arch, Void, Mint, etc?